plugins/doc_fragments/alicloud.py

```diff
@@ -11,46 +11,96 @@ class ModuleDocFragment(object):
+    type: str
+  alicloud_assume_role:
+    description:
+      - If provided with a role ARN, Ansible will attempt to assume this role using the supplied credentials.
+      - The nested assume_role block supports I(alicloud_assume_role_arn), I(alicloud_assume_role_session_name),
+        I(alicloud_assume_role_session_expiration) and I(alicloud_assume_role_policy)
+    type: dict
+    aliases: ['assume_role']
+  alicloud_assume_role_arn:
+    description:
+      - The Alibaba Cloud role_arn. The ARN of the role to assume. If ARN is set to an empty string,
+        it does not perform role switching. It supports environment variable ALICLOUD_ASSUME_ROLE_ARN.
+        ansible will execute with provided credentials.
+    aliases: ['assume_role_arn']
+    type: str
+  alicloud_assume_role_session_name:
+    description:
+      - The Alibaba Cloud session_name. The session name to use when assuming the role. If omitted,
+        'ansible' is passed to the AssumeRole call as session name. It supports environment variable
+        ALICLOUD_ASSUME_ROLE_SESSION_NAME
+    aliases: ['assume_role_session_name']
+    type: str
+  alicloud_assume_role_session_expiration:
+    description:
+      - The Alibaba Cloud session_expiration. The time after which the established session for assuming
+        role expires. Valid value range 900-3600 seconds. Default to 3600 (in this case Alicloud use own default
+        value). It supports environment variable ALICLOUD_ASSUME_ROLE_SESSION_EXPIRATION
+    aliases: ['assume_role_session_expiration']
+    type: int
+  ecs_role_name:
+    description:
+      - The RAM Role Name attached on a ECS instance for API operations. You can retrieve this from the 'Access Control'
+        section of the Alibaba Cloud console.
+      - If you're running Ansible from an ECS instance with RAM Instance using RAM Role, Ansible will just access the
+        metadata U(http://100.100.100.200/latest/meta-data/ram/security-credentials/<ecs_role_name>) to obtain the STS
+        credential. This is a preferred approach over any other when running in ECS as you can avoid hard coding
+        credentials. Instead these are leased on-the-fly by Ansible which reduces the chance of leakage.
+    aliases: ['role_name']
+    type: str
+  profile:
+    description:
+      - This is the Alicloud profile name as set in the shared credentials file. It can also be sourced from the
+        ALICLOUD_PROFILE environment variable.
+    type: str
+  shared_credentials_file:
+    description:
+      - This is the path to the shared credentials file. It can also be sourced from the ALICLOUD_SHARED_CREDENTIALS_FILE
+        environment variable.
+      - If this is not set and a profile is specified,  ~/.aliyun/config.json will be used.
     type: str
-    aliases: [ security_token ]
 author:
-- He Guimin (@xiaozhu36)
+    - "He Guimin (@xiaozhu36)"
 requirements:
-- python >= 2.6
+    - "python >= 3.6"
 extends_documentation_fragment:
```

Comment:
Should these two lines be here?
Looks like this documentation fragment is loading itself.

Reply:
I deleted the specified two lines, check will report an error, I do not known why, in view of the previous code has always existed, I think it can be retained.